docs/usage/getting-started/use-cases.md

```diff
@@ -0,0 +1,135 @@
+# Use Cases
+
+This page describes some common use cases for Renovate, for those who are new and would like to get their heads around the capabilities.
+
+## Development Dependency Updates
+
+The original use case, and still the most popular one, is for developers to automate updating of their development dependencies.
```

Comment:
When reading this, it seems like Renovate might not update _production_ dependencies, but maybe that's me being silly. 

Readers might think: "Oh, I'll have to update my _production_ dependencies myself then, as Renovate only updates _development_ dependencies.